virttest/qemu_storage.py

```diff
@@ -1096,6 +1096,128 @@ def snapshot_apply(self):
 
         process.run(cmd)
 
+    def bitmap_add(self):
+        """
+        Add a bitmap to a image.
+
+        :note: params should contain:
+               bitmap_name -- the name of bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --add -g %s -f %s %s %s" % (
```

Comment:
Do we need to consider --object as well as the image filename represented in json for the network storage? If that's not considered currently, we may also need to find a way to extend the support easily, please refer to the create_cmd = () or check_cmd = (), which includes most options.

Reply:
Bitmap is mostly used for incremental backup feature, and here it is working as a sub option of qemu-img which is mostly for checking the bitmap info or some simple operations. And I believe that the basic tests for the qcow2 file images are enough.